### PR TITLE
Fixed: Two scroll bars appear on small screens on almost every page

### LINKED ISSUE #1308: [🕹️] Two scroll bars appear on small screens on almost every page
### What side quest or challenge are you solving?

latestz

### Points

@mfts ?

### Description

_No response_

### Provide proof that you've completed the task

PR

### PR DESCRIPTION
**Pull request by AshishViradiya153**
fixes #1308

### CODE REVIEW

File: components/layouts/app.tsx [hunk 1]
```diff
@@ -8,7 +8,7 @@ export default function AppLayout({ children }: { children: React.ReactNode }) {
       <div className="h-dvh flex-1">
         {/* Trial banner shown only on trial */}
         <TrialBanner />
-        <main className="h-dvh flex-1 lg:p-2">
+        <main className="h-[calc(100dvh-60px)] flex-1 lg:h-dvh lg:p-2">
```
Comment (mfts) [inline components/layouts/app.tsx:11]: where does this 60px come from?

**Review by mfts — COMMENTED** [review]
just a quick follow up how you arrived at this classname ?

File: components/layouts/app.tsx (hunk 1 above)
Comment (AshishViradiya153) [reply components/layouts/app.tsx:11]: navbar height!

**Comment by AshishViradiya153** [comment]
recheck

**Review by mfts — APPROVED** [review]

**Pull request merged**